website/docs/wallet/exiting-a-validator.md

```diff
@@ -11,18 +11,26 @@ import {HeaderBadgesWidget} from '@site/src/components/HeaderBadgesWidget.js';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-Voluntarily exiting your validator from the Ethereum network is a one time command using the [prysmctl tool](../prysm-usage/prysmctl.md). **note:** previously uses a command under the validator client, and can still be accessed that way.
 
- 1. Ensure access to a fully synced beacon node. 
- 1. Issue the `validator exit` command to your validator and providing access to your validator keys through the `--wallet-dir` flag or [web3signer](web3signer.md) and the `--beacon-rpc-provider`flag to the synced beacon node (examples provided below).
- 2. Select the account(s) that should be exited. This step can be skipped by specifying the account(s) via the `--public-keys` flag when issuing the `validator exit` command.
- 3. Confirm your understanding of the consequences of exiting your validator by typing `Exit my validator` when prompted.
+:::caution Public Preview
+
+**This document is currently in public preview** and may change significantly as we receive feedback from users like you. Join our [Discord server](https://discord.gg/prysmaticlabs) or click the `Request an Update` button above to share your feedback.
+
+:::
+
+
+Voluntarily exiting your validator from the Ethereum network is a one time command using the [prysmctl tool](../prysm-usage/prysmctl.md). Note that this operation was previously facilitated by a command exposed by the Prysm validator client, and can still be accessed that way. At a high level, you'll follow these steps to exit your validator:
+
+ 1. Ensure that you have access to a fully synced beacon node.
+ 2. Issue the `validator exit` command to your validator and allow the beacon node to access to your validator keys through the `--wallet-dir` flag or [web3signer](web3signer.md) and the `--beacon-rpc-provider` flag (examples provided below).
+ 3. Select the account(s) that should be exited. This step can be skipped by specifying the account(s) via the `--public-keys` flag when issuing the `validator exit` command.
+ 4. Confirm your understanding of the consequences of exiting your validator by typing `Exit my validator` when prompted.
 
 After providing confirmation, voluntary exit request will be broadcasted through your beacon node. Visit our [Command-line options documentation](../prysm-usage/parameters.md) for more configuration options.
 
 :::caution 
 
-Although validator nodes can voluntarily exit, funds won't be withdrawable until after Capella/Shanghai hardfork and the bls-to-exeuction-change command is available. Learn more on how to withdraw earnings or fully withdraw your validator once available in [our guide](withdraw-validator.md)
+Although validator nodes can voluntarily exit, funds won't be withdrawable until after Capella/Shanghai hardfork and the `bls-to-exeuction-change` command is available. Learn more on how to withdraw earnings or fully withdraw your validator once available in [our guide](withdraw-validator.md)
```

Comment:
```suggestion
Although validator nodes can voluntarily exit, funds won't be withdrawable until after Capella/Shanghai hardfork and the `bls-to-exeuction-change` command is available. Learn more on how to withdraw earnings or fully withdraw your validator once available in [our guide](withdraw-validator.md)

validator-edit command only supports gRPC meaning the beacon-rpc-provider must be a prysm beacon node to utilize
```

Reply:
@james-prysm I'd recommend "The `validator-exit` command only supports gRPC, which means that the specified `beacon-rpc-provider` needs to be a Prysm beacon node (because Prysm's beacon node client is the only client that supports gRPC)."

Meta -> complete sentences, code formatting around commands for consistency, capitalizing proper nouns, providing minimum-viable explanation.